### ISSUE TITLE
Intl support

### ISSUE THREAD

**Issue by sassomedia** [amcharts/amcharts5#23]
I'm in the process of converting am4 to am5 charts and one thing we heavily depend on is `Intl` support. I don't see this currently documented or supported in am5 and when I try something equivalent to the code in am4:
```javascript
chart.dateFormatter.dateFormat = { month: "short", day: "numeric" };
chart.dateFormatter.intlLocales = userLocale;
```
none of this works in am5:
```javascript
root.dateFormatter.set("dateFormat", { month: "short", day: "numeric" }); // "dateFormat" only supports Strings
root.dateFormatter.set("intlLocales", userLocale); // "intlLocales" is not an available setting on "dateFormatter"
root.locale = am5locales_en_US; // this seems to only be for manually imported locales
```

I hope I'm missing something or maybe it's just not part of the beta but will be in the release version, yes?

**Comment by martynasma** [comment]
We don't have Intl support, yet. But we plan to add it. Will keep you posted.

**Comment by sassomedia** [comment]
Sounds good, thanks for the confirmation!

**Comment by martynasma** [comment]
Implemented in [5.0.0-beta.29](https://github.com/amcharts/amcharts5/releases/tag/5.0.0-beta.29).

## [5.0.0-beta.29] - 2021-10-01

### Added
- Wherever date or number format can be set (`DateFormatter`, `NumberFormatter`, `DateAxis`, `ValueAxis`, `Exporting`) it can now be set as `Intl.DateTimeFormatOptions` (for date format) or `Intl.NumberFormatOptions` (for number format) object in additional to string-based format. [More info](https://www.amcharts.com/docs/v5/tutorials/formatting-date-time-and-numbers-using-intl-object/).
- New list type `ListAutoDispose` added.

### Changed
- `XYCursor` will now ignore mouse movement if chart is obstructed by other element, unless zoomin/panning.
- `series`, `xAxes`, and `yAxes` on an `XYChart` are now `ListAutoDispose`, which means that removing items from those lists will automatically dispose them. Set `autoDispose = false` on those lists if you are planning on removing and reusing items.

### Fixed
- `Sunburst` nodes were shown on drill-down when they were not supposed to in some cases.
- `Hierarchy` node elements were ignoring `templateField`.
- It was impossible to move `XYChart` scrollbars to some other container.
- Fixed memory leaking when scrolling `LineSeries`, and when disposing series or other items.
- `XYCursor` was not firing `selectended` event if `behavior` was set to `"selectX"`, `"selectY"`, or `"selectXY"`.
- `XYCursor` was hiding selection when when behavior was set to `"selectX"`, `"selectY"`, or `"selectXY"`.
- `SerialChart.series.clear()` was not working properly.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download-v5/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by sassomedia** [comment]
Awesome, that was fast! Just tried it, works great! 👍
